### PR TITLE
Update details of Device-specific private key

### LINKED ISSUE #3548: Device specific private key is now supported for 2712
There's a comment in here https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#device-specific-private-key
that says "Device specific private key" is not implemented for 2712. This is now fixed. The only difference is that there are 16 rows for 2712.

### PR DESCRIPTION
**Pull request by peterharperuk**
This has been implemented for 2712 where there are now 16 rows.

Update the links because the script no longer lives in usbboot but comes from rpi-eprom which is now a submodule in usbboot.

Fixes #3548

### CODE REVIEW

**Comment by peterharperuk** [comment]
@timg236

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:178]:
```suggestion
NOTE: The https://github.com/raspberrypi/usbboot[`usbboot`] repository contains all the tools you need, including https://github.com/raspberrypi/rpi-eeprom[`rpi-eeprom`] as a Git submodule.
```
Suggestion: Git uses 'submodule' as a single word.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:192]:
```suggestion
Writes a 32-byte randomly generated number to the device private key.

```
No need for the trailing `+` here; it actually prevents that warning from displaying as an admonition.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:193]:
```suggestion
WARNING: This operation cannot be undone.

```

For cleanliness' sake, we like to put newlines in between page elements.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:199]:
```suggestion
NOTE: To specify the number of OTP rows to use, pass `-l <word count>`. To specify a start location in the key store, pass `-o <word offset>`.
```
Suggestion: rephrase this to begin with the purpose, then explain the method.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:164]:
```suggestion
Devices that use the Broadcom BCM2712 processor have 16 rows of OTP data (512 bits) to support filesystem encryption.
Devices that do not use BCM2712 have 8 rows of OTP (256 bits) available for use as a device-specific private key.
```
Suggestion: make it clearer that we're referring to the Broadcom BCM2712 processor.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:176]:
```suggestion
The https://github.com/raspberrypi/rpi-eeprom/blob/master/tools/rpi-otp-private-key[`rpi-otp-private-key`] script wraps the device private key `vcmailbox` APIs to make it easier to read and write a key in the OpenSSL format.
```

Suggestion: minor copy improvement.

**Review by nathan-contino — APPROVED** [review]
LGTM, as long as we implement my suggestions :)

**Comment by nathan-contino** [comment]
@peterharperuk should I merge now, or would you rather wait for a technical review from @timg236

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (lurch) [reply documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc]: " wraps the necessary `vcmailbox` APIs" ?

Comment (lurch) [reply documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc]: In fact I guess you could even link this to the `==== Mailbox API for reading/writing the key.` section?

**Comment by peterharperuk** [comment]
I'm happy to get this merged now. Thanks a lot for the updates!

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc]: _technically_ we could release a future device with a processor older than BCM2712, so I don't know if this necessarily holds up. Maybe just "Raspberry Pi 5" and "prior to Raspberry Pi 5"? But there are enough specifics here related to the CPU that this is presumably subject to change in future processors.


...definitely worth a revisit in the future when the situation changes. For now, I'm happy enough with it.

**Comment by nathan-contino** [comment]
Merged. Thank YOU for the PR!

**Pull request merged**
